docs/changelog.rst

```diff
@@ -16,6 +16,11 @@ Core API
 - Start ``tlid`` counting at 1 instead of 0 to keep in sync with MPD's
   ``songid``.
 
+- `get_time_position` now returns the seek target while a seek is in progress.
```

Comment:
Either you need double backtick quotes or use `:meth:~mopidy.core.PlaybackController.get_time_position`.